contracts/SpokePool.sol

```diff
@@ -1062,6 +1165,7 @@ abstract contract SpokePool is
             }
         }
 
+        // TODO: Add in some limit to `fillAmount` to mitigate censor attacks on the fill counter.
```

Comment:
Note: in other places, we typically do this by requiring that the `amount` value passed into any of the fill (or deposit) methods is below some value rather than putting a check in this function.